providers/cephfs.rb

```diff
@@ -29,6 +29,7 @@ def manage_mount(directory, subdir, use_fuse, action)
       dump 0
       pass 0
       action action
+      not_if "mount | grep \"^ceph-fuse on #{Regexp.escape(directory)}\"" if action == :mount
```

Comment:
The chef [s3fs mount cookbook](https://github.com/hw-cookbooks/s3fs-fuse/blob/develop/recipes/default.rb) checks where the directory is a mountpoint, not specifically that it is an s3fs mountpoint.